website/docs/examples/pendant_state/README.md

```diff
@@ -0,0 +1,135 @@
+---
+id: pendant-state
+title: Pendant State
+---
+
+# Amiga Pendant State Example
+
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
+like `functions`, `loops`, and `classes`, since the example utilizes these fundamentals.
+2. [**farm-ng Canbus Service Overview**](/docs/concepts/canbus_service):
+This overview provides a base understanding of the gRPC service the client you create will connect to.
+:::
+
+The [**Amiga Pendant State**](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/pendant_state/main.py)
+example is a basic way of showing how to access and decode the
+`PendantState` values streamed by the canbus service.
+
+You can either run this example directly on a brain by `ssh`'ing in, or use your local PC.
+If using your local PC, it should be either connected to the same local network as the brain
+or linked to it through tailscale.
+
+The requirements to run this example are to have a
+[**farm-ng brain**](/docs/brain/) running the `canbus service`, with a pendant attached to your Amiga.
+There will be no `/pendant` stream if your amiga does not have a wired pendant connected.
+
+At the current dashboard firmware release `v0.4.0`, the `PendantState` is not published on the CAN
+bus for pendant alternatives, including Kartech wireless remotes and the on-screen pendant.
+
+## `PendantState` details
+
+:::tip
+Check out [`farm-ng-amiga/py/farm_ng/canbus/packet.py`](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/farm_ng/canbus/packet.py)
+or the auto-generated [**`canbus.packet` API docs**](/docs/reference/brain/canbus/packet) for the latest.
+for the latest
+:::
+
+- `x`: Joystick `[left, right]`, on range `[-1.0, 1.0]`
+- `y`: Joystick `[reverse, forward]`, on range `[-1.0, 1.0]`
+- `buttons`: Bit masked integer representing which buttons are
```

Comment:
It looks like something is missing here...
Maybe "which buttons are pressed"?

Reply:
Thanks, good catches